chemprop/nn/predictors.py

```diff
@@ -244,7 +244,7 @@ def forward(self, Z: Tensor) -> Tensor:
     def train_step(self, Z: Tensor) -> Tensor:
         Y = super().forward(Z)
 
-        F.softplus(Y) + 1
+        return F.softplus(Y) + 1
```

Comment:
Interesting that we didn't catch this earlier. This reminds me that we need to add a test for dirichlet. It doesn't need to be added here in this PR.

Reply:
It is probably because we haven't extensively tested the more advanced model settings yet, so no one found this bug. Also, I would be happy to include more comprehensive integration tests after this PR.